### PR TITLE
Device Patient Access Updated

### PR DESCRIPTION
**Pull request by NimishaMN**
Description
----
Introducing Device Patient Access

Screenshot 2020-12-16 at 11 35 11 AM
Screenshot 2020-12-16 at 12 33 32 PM

Screenshot 2020-12-16 at 11 35 37 AM
Screenshot 2020-12-16 at 11 35 52 AM

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: lib/resources/example_json/r4_examples_device.rb [hunk 1]
```diff
@@ -81,5 +81,175 @@ module Resources
...
+
+    R4_DEVICE_PATIENT_BUNDLE ||= {
+      "resourceType": "Bundle",
+      "id": "93c4fee8-8b83-4124-bb24-2b353a4cc71b",
+      "type": "searchset",
+      "total": 1,
+      "link": [
+          {
+              "relation": "self",
+              "url": "http://localhost:3000/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Device?patient=12724066"
```
Comment (ashu-bdvt) [inline lib/resources/example_json/r4_examples_device.rb:184]: Update url to real endpoint and not localhost

File: lib/resources/example_json/r4_examples_device.rb [hunk 2]
```diff
@@ -81,5 +81,175 @@ module Resources
...
+          }
+      ],
+      "entry": [
+          {
+              "fullUrl": "http://localhost:3000/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Device/31539245",
```
Comment (ashu-bdvt) [inline lib/resources/example_json/r4_examples_device.rb:189]: Update url to real endpoint and not localhost

**Review by ashu-bdvt — CHANGES_REQUESTED** [review]
Remove Jira reference from the PR title. 'R4 Device Patient Access Supported' should be sufficient

File: lib/resources/example_json/r4_examples_device.rb (hunk 1 above)
Comment (NimishaMN) [reply lib/resources/example_json/r4_examples_device.rb]: Addressed. https://github.com/cerner/fhir.cerner.com/pull/528/commits/606b3510fe05a22850556bf15677e2df4201c24e#diff-18a5da189260e681dfa7678232f23174b85ab32d492019976586b873f3b5d802R184

File: lib/resources/example_json/r4_examples_device.rb (hunk 2 above)
Comment (NimishaMN) [reply lib/resources/example_json/r4_examples_device.rb]: Addressed. https://github.com/cerner/fhir.cerner.com/pull/528/commits/606b3510fe05a22850556bf15677e2df4201c24e#diff-18a5da189260e681dfa7678232f23174b85ab32d492019976586b873f3b5d802R189

**Review by ashu-bdvt — APPROVED** [review]

**Review by aki-mathur — APPROVED** [review]

**Comment by ebbp123** [comment]
Verified live:
Screen Shot 2020-12-17 at 4 14 51 PM

**Pull request merged**